src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -3023,7 +3023,7 @@ def webapp_up(cmd, name, resource_group_name=None, plan=None, location=None, sku
         site_config = client.web_apps.get_configuration(rg_name, name)
     else:  # need to create new app, check if we need to use default RG or use user entered values
         logger.warning("webapp %s doesn't exist", name)
-        sku = get_sku_to_use(src_dir, sku)
+        sku = get_sku_to_use(src_dir, html, sku)
```

Comment:
can we avoid using positional arguments for each function?

Reply:
we need to pass these in case of the flag being set

Reply:
I know. I will prefer something like get_sku_to_use(src_dir=src_dir, html=html, sku=sku) to avoid positional arguments